runtime/CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/
 )
```

Comment:
Are you going to change the location of the header and `*.inl` to `include/legion` as well?

Reply:
@streichler is working on a change to do this in a different branch (as it requires additional changes). Let's revisit this after that change has landed.

Reply:
Ok, but it should be ${CMAKE_INSTALL_INCLUDEDIR}` instead of `include` anyhow.